### DISCUSSION TITLE
2-Column layout

### DISCUSSION THREAD

**Discussion by rima1098** [QuestPDF/QuestPDF#829] [Q&A] [answered]
Hi, 

Is require a 2-column layout for our documents and i'm curious if there is a functionality already to do this automatically. I need content to wrap from the first column to the second column and then to the next page, something like this:

Content 1 |  Content 1
Content 1 |  Content 2
Content 1 |  Content 2
----- Page break -----
Content 2 |  Content 3
Content 2 |  Content 3
Content 3 |  

I know that i could manually place Content inside two columns and do the page breaks manually since the content is static but it would be more comfortable to just add the whole content and do the pagebreaks and column wraps automatically. Is there any way to do this?

**Comment by DawidRobertNowak** [comment, marked as answer]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

This is now supported via the [`MultiColumn`](https://www.questpdf.com/api-reference/multi-column.html) element, which arranges content into newspaper-style columns and flows it automatically — filling the first column, wrapping into the next, and continuing onto the following page, with no manual breaks. Use `Columns(2)` for a two-column layout and `Spacing(...)` for the gap between columns; optionally, `Spacer()` adds a divider between columns and `BalanceHeight()` evens out the column heights.

Should you encounter any further problems, please don't hesitate to open a new issue.
Thank you!
